Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2077,6 +2078,16 @@ void RenderStyle::setFontSize(float size)
     fontCascade().update(currentFontSelector);
 }
 
+void RenderStyle::setFontSizeAdjust(std::optional<float> sizeAdjust)
+{
+    FontSelector* currentFontSelector = fontCascade().fontSelector();
```

Comment:
`auto* selector` would be better

Reply:
Bonus points if you fix up the sibling functions above/below to do the same.